[v0.24.0] `yarn rw test web` is Failing

## How to Reproduce
- Run through the E2E process to create a new project and add code per E2E.
- Enter the new project and run `yarn rw test web --no-watch`

Results in the following errors:
```console
 FAIL   web  web/src/pages/HomePage/HomePage.test.js
  ● Test suite failed to run

    Cannot find module '@redwoodjs/router/dist/index' from '../node_modules/@redwoodjs/testing/dist/MockRouter.js'

    Require stack:
      /Users/price/Repos/xx-delete/node_modules/@redwoodjs/testing/dist/MockRouter.js
      src/Routes.js
      /Users/price/Repos/xx-delete/node_modules/@redwoodjs/testing/dist/MockProviders.js
      /Users/price/Repos/xx-delete/node_modules/@redwoodjs/testing/dist/customRender.js
      /Users/price/Repos/xx-delete/node_modules/@redwoodjs/testing/dist/index.js
      /Users/price/Repos/xx-delete/node_modules/@redwoodjs/core/dist/configs/browser/jest.setup.js

      at Resolver.resolveModule (../node_modules/jest-resolve/build/index.js:306:11)
      at Object.<anonymous> (../node_modules/@redwoodjs/testing/dist/MockRouter.js:31:14)
```

--- a/packages/testing/package.json
+++ b/packages/testing/package.json
@@ -9,6 +9,7 @@
   "types": "dist/index.d.ts",
   "license": "MIT",
   "dependencies": {
+    "@redwoodjs/auth": "0.23.0",
     "@redwoodjs/internal": "^0.23.0",
     "@redwoodjs/web": "^0.23.0",
     "@testing-library/react": "11.2.2",

--- a/packages/testing/tsconfig.json
+++ b/packages/testing/tsconfig.json
@@ -14,6 +14,7 @@
   ],
   "references": [
     { "path": "../internal" },
-    { "path": "../web" }
+    { "path": "../web" },
+    { "path": "../auth" }
   ]
 }

